README.md

```diff
@@ -8,6 +8,21 @@ This setup allows for a Node/Express/React app which can be easily deployed to H
 
 The front-end React app will auto-reload as it's updated via webpack dev server, and the backend Express app will auto-reload independently with nodemon.
 
+## Wireframes
+
+### Home Page
+<img src ="Wireframe_Images/Podcast-Wireframe.png" width="400">
+
+### Login Page
+<img src="Wireframe_Images/Podcast-Wireframe3.png" width="400">
+
+### Favorites
+<img src= "Wireframe_Images/Podcast-Wireframe4.png" width="400">
+
+### Subscribed
+<img src= "Wireframe_Images/Podcast-Wireframe5.png" width="400">
+
+
```

Comment:
The source for these images might have to change once it is up on GitHub. Make sure you change it appropriatley.

Reply:
i believe github supports this html for the images when done this way

Reply:
okay, cool. In the past I have always had to copy the exact URL link from GitHub for the image once it was uploaded.

Reply:
Just checked the README on your branch. Cool! I will need to do it that way from now on.